Jenkinsfile

```diff
@@ -1,11 +1,46 @@
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '10')),
+    parameters([
+        booleanParam(defaultValue: false,
+            description: 'Override distfeeds.conf links',
+            name: 'OVERRIDE_PACKAGES'),
+        stringParam(defaultValue: '',
+            description: 'Works only with OVERRIDE_PACKAGES set to true.\
+                          Path to a directory containing the packages folder,\
+                          if null defaults to ${JOB_PATH}/artifact/bin/pistachio',
+            name: "PACKAGES_PATH"),
+        stringParam(defaultValue: '',
+            description: 'Path to the job that needs to be tested, if null downloads newest stable release',
+            name: "JOB_PATH"),
+    ])
+])
+
 node('boardfarm') {
     stage("Configure") {
         deleteDir()
 
+        def image_path = "http://downloads.creatordev.io/openwrt/ci40-v1.1.0/pistachio/marduk"
+        def image_name = "openwrt-v1.1.0-pistachio-marduk-squashfs-factory.ubi"
```

Comment:
I can redo the JOB_PATH as IMAGE_PATH then and move the responsibility of finding out what image to pass to a job kicking this one.